### DISCUSSION TITLE
Who won Best Actor 🎥🏆in 2014?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#763] [Galaxy Trivia] [answered]
**Bonus:** For a role in which movie?

**Comment by peterbaccaro** [comment]
The Best Actor award for the year 2014 was won by **Matthew McConaughey** for his performance in the movie "Dallas Buyers Club".

**Comment by SANKARAMDAS** [comment, marked as answer]
Matthew McConaughey
The Best Actor winner for "Dallas Buyers Club" cuts up in the audience.
